variables.tf

```diff
@@ -68,8 +68,13 @@ variable "docker_version" {
   type        = "string"
 }
 
+variable "ct_common_config_yaml" {
+  description = "Common Circus Train Config for internal Circus Train instance. Refer to [Circus-Train README](https://github.com/HotelsDotCom/circus-train/blob/master/README.md) for more details."
```

Comment:
Would this show up as a clickable link? Because I thought it happened only in `.md` files. 

If it doesn't show up as a link, it could simply be "Circus Train README (https://github.com/HotelsDotCom/circus-train/blob/master/README.md)"

(also Circus Train doesn't have a dash in the middle 🙂)

Reply:
No it won't but its necessary to keep it like this here so that I can use `terraform-docs` markdown documentation generator tool to generate the table that you see in the README file.

https://github.com/segmentio/terraform-docs

Reply:
also didn't know :)

Reply:
Yeah. its a useful little tool.

```
brew install terrform-docs
terraform-docs markdown . (in the terraform folder with variables.tf)
```